etl/tests/test_database.py

```diff
@@ -22,5 +22,41 @@ def test_load(database_coordinates: database.DatabaseCoordinates,
               database_name: str,
               collection: str,
               load_data: typing.List[dwelling.Dwelling]):
-    database.load(database_coordinates, database_name, collection, load_data)
+    database.load(
+        coords=database_coordinates,
+        database_name=database_name,
+        collection_name=collection,
+        data=load_data,
+        update=False
+    )
+    assert mongo_client[database_name][collection].count() == 3
+
+
+def test_load_update(database_coordinates: database.DatabaseCoordinates,
+                     mongo_client: pymongo.MongoClient,
+                     database_name: str,
+                     collection: str,
+                     load_data: typing.List[dwelling.Dwelling]):
+    database.load(
+        coords=database_coordinates,
+        database_name=database_name,
+        collection_name=collection,
+        data=load_data,
+        update=False
+    )
     assert mongo_client[database_name][collection].count() == 3
+    load_data[0] = dwelling.Dwelling(house_id=1, year_built=2001, city='Ottawa', region=dwelling.Region.ONTARIO,
+                                     forward_sortation_area='K1P', evaluations=[])
+    load_data.append(
+        dwelling.Dwelling(house_id=4, year_built=2001, city='Ottawa', region=dwelling.Region.ONTARIO,
+                          forward_sortation_area='K1P', evaluations=[])
+    )
+
+    database.load(
+        coords=database_coordinates,
+        database_name=database_name,
+        collection_name=collection,
+        data=load_data,
+        update=True
+    )
+    assert mongo_client[database_name][collection].count() == 4
```

Comment:
should we check that house_id 1  changed appropriately?

Reply:
Probably, I'll add that.